app/konnect/dev-portal/applications/enable-app-reg.md

```diff
@@ -4,16 +4,30 @@ content_type: how-to
 ---
 
 To grant developers access to [register an application](/konnect/dev-portal/applications/dev-reg-app-service), you must enable application registration for a service version.
-When you enable application registration, {{site.konnect_saas}} enables two plugins automatically: [ACL](/hub/kong-inc/acl), and your choice of [Key Authentication](/hub/kong-inc/key-auth)
-or [OIDC](/hub/kong-inc/openid-connect). These plugins run in the background to support application registration for the service and are managed by
+When you enable application registration, {{site.konnect_saas}} enables plugins automatically to support the desired mode, either key authentication or OpenID Connect.
```

Comment:
Nit:
```suggestion
When you enable application registration, {{site.konnect_saas}} automatically enables plugins to support the desired mode, either [Key Authentication](https://docs.konghq.com/hub/kong-inc/key-auth/) with [ACL](https://docs.konghq.com/hub/kong-inc/acl/) or [OpenID Connect](https://docs.konghq.com/hub/kong-inc/openid-connect/).
```

I just want to be more explicit in the explanations, if that makes sense.

Reply:
Please disregard!